Kernel/Syscalls/resource.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2022, Lucas Chollet <lucas.chollet@free.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Kernel/Process.h>
+
+namespace Kernel {
+
+ErrorOr<FlatPtr> Process::sys$getrusage(int who, Userspace<rusage*> user_usage)
+{
+    VERIFY_PROCESS_BIG_LOCK_ACQUIRED(this)
+
+    rusage usage;
```

Comment:
You will need to zero initialize this after you fix the other comment about having a C++ constructor in a C header.
```suggestion
    rusage usage {};
```